transit_odp/pipelines/tests/test_dataset_etl/test_servicepattern.py

```diff
@@ -97,3 +102,87 @@ def test_servicepattern(self):
                 break
 
         self.assertTrue(found)
+
+
+columns_lines = [
+    "line_id",
+    "line_name",
+    "outbound_description",
+    "inbound_description",
+]
+
+columns_service_patterns = [
+    "service_code",
+    "direction",
+    "geometry",
+    "localities",
+    "admin_area_codes",
+    "description",
+    "line_name",
+]
+
+
+@override_flag("is_timetable_visualiser_active", active=True)
+class ETLServicePatterns(ExtractBaseTestCase):
```

Comment:
Can we add a test case that checks both lines in the same service is stored as expected

Reply:
Already handled with test case - test_correct_description_based_on_direction_and_line